index.html

```diff
@@ -41,8 +41,122 @@ <h3 class="link-1"> <a href="#"> LET'S CONNECT</a></h3>
                    </ul>
                 </div>
             </footer>
+        </section> 
+        </div>
+        <section>
+            <div class="card">
+                <img class="Snapshoot" src="./image/Snapshoot-Portfolio.png" alt="">
+                <h1 class = "title">Portfolio</h1>
+                <ul class="frame">
+                    <li class="detail">CANOPY</li>
+                    <li class="detail">BACK END DEV</li>       
+                    <li class="detail">2015</li>       
+                </ul>
+                <p class="detailp">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <div class="tags">
+                    <div class="tag"><h6 class="tag-text">html</h6></div>
+                    <div class="tag"><h6  class="tag-text">css</h6></div>
+                    <div class="tag"><h6  class="tag-text">javascript</h6></div>
+                </div>
+                <button type="submit" class="normal-button">See Project</button>
```

Comment:
- [ ] Currently, your button doesn't have a hover effect as specified in the design, kindly check the image below for the effect expected on your buttons: 
![image](https://user-images.githubusercontent.com/61408860/165708419-59739ca1-e612-4503-9081-584b49b7a8f0.png)